Purchases/Attribution/AttributionFetcher.swift

```diff
@@ -138,4 +134,56 @@ private extension AttributionFetcher {
         }
     }
 
+    @available(iOS 14.0.0, tvOS 14.0.0, *)
```

Comment:
I broke apart this method into many discrete steps so I could reuse the pieces across both `isAuthorizedToPostSearchAdsInATTRequiredOS` and `fetchAuthorizationStatus`